.github/workflows/build.yml

```diff
@@ -35,3 +35,10 @@ jobs:
         with:
           name: 'backstop_test_results'
           path: '${{ github.workspace }}/backstop/results/'
+
+      - name: Host test results on gh pages
+        uses: peaceiris/actions-gh-pages@v3.7.3
```

Comment:
I think we should use this one: https://github.com/JamesIves/github-pages-deploy-action

It's officially recommended by github, has fewer issues, and is more actively maintained 
https://github.blog/2020-09-25-github-action-hero-james-ives-and-github-pages-deploy/